discovery/src/main/java/com/ibm/watson/developer_cloud/discovery/v1/model/DeleteDocumentResponse.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2017 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.ibm.watson.developer_cloud.discovery.v1.model;
+
+import com.google.gson.annotations.SerializedName;
+import com.ibm.watson.developer_cloud.service.model.GenericModel;
+
+/**
+ * DeleteDocumentResponse.
+ */
+public class DeleteDocumentResponse extends GenericModel {
+
+  /** The unique identifier of the document. */
+  @SerializedName("document_id")
+  private String documentId;
+  /** Status of the document. A deleted document has the status deleted. */
+  private String status;
```

Comment:
We can generate enums but I think we were not going to do it because status could be enhanced and adding a new status is not a breaking change in the API but it's in the SDK.

We have a similar problem with `models` in Speech to Text where there are a set of predefined models like `narrowBand_US` or `broadband_US` but the Speech team keeps adding new models